tests/Feature/Monitor.php

```diff
@@ -105,30 +105,40 @@ public function testMonitorAPI() : void
         ]);
         $response_json = $response->json();
         $this::assertTrue(array_key_exists('time_chart_data', $response_json));
-        $this::assertEquals($response_json['time_chart_data'][0]['key'], 'success');
-        $this::assertEquals($response_json['time_chart_data'][0]['color'], ClassicPalette::Success->value);
-        $success_values = $response_json['time_chart_data'][0]['values'];
-        $last_success_value = $success_values[array_key_last($success_values)][1];
-        $this::assertEquals($last_success_value, 1);
-
-        $this::assertEquals($response_json['time_chart_data'][1]['key'], 'fail');
-        $this::assertEquals($response_json['time_chart_data'][1]['color'], ClassicPalette::Failure->value);
-        $fail_values = $response_json['time_chart_data'][1]['values'];
-        $last_fail_value = $fail_values[array_key_last($fail_values)][1];
-        $this::assertEquals($last_fail_value, 1);
+        $this::assertTrue(array_key_exists('data', $response_json['time_chart_data']));
```

Comment:
Check out the docs here: https://laravel.com/docs/10.x/http-tests#testing-json-apis

Laravel provides a set of JSON testing functions which can do exactly this.

Reply:
Cool, I'll update the test to use these instead